src/HealthChecks.AzureServiceBus/Configuration/AzureServiceBusQueueHealthCheckOptions.cs

```diff
@@ -20,6 +20,7 @@ public class AzureServiceBusQueueHealthCheckOptions : AzureServiceBusHealthCheck
     /// is used set this to <see langword="false"/>. By default <see langword="true"/>.
     /// </remarks>
     public bool UsePeekMode { get; set; } = true;
+    public bool IsExceptionDetailsRequired { get; set; } = true;
```

Comment:
```suggestion
    public bool CaptureException { get; set; } = true;
```

It seems that such option is applicable to every health check into solution.

Reply:
After thinking about that a bit I'm sure this option is useless.
1. This behaviour is not specific to that HC. Such option is applicable to every HC.
2. By default ASP.NET Core captures exception but it writes plain text response with only status code, see ` public Func<HttpContext, HealthReport, Task> ResponseWriter { get; set; } = HealthCheckResponseWriters.WriteMinimalPlaintext;
`
3. One can set custom `ResponseWriter`. Indeed this repo does, see `UIResponseWriter.WriteHealthCheckUIResponse`.
4. Also #1581 provides a feture to hide exception text.
5. You may filter exceptions in ` public static UIHealthReport UIHealthReport.CreateFrom(HealthReport report, Func<Exception, string>? exceptionMessage = null)`.

Reply:
@sungam3r  is https://github.com/Xabaril/AspNetCore.Diagnostics.HealthChecks/pull/1581 present in version 6 of the package?

Reply:
Closing PR. Will wait for the feature  in 7.x.x release.